inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -17,6 +17,10 @@ void LayerTestsCommon::Run() {
     LoadNetwork();
     Infer();
     Validate();
+
+    if (targetDevice == std::string{CommonTestUtils::DEVICE_GPU}) {
```

Comment:
These changes is in destructor. Please remove

Reply:
in destructor it is not correct place